docker-compose-setup/init.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+cd "$(dirname "$0")" || exit
+export $(cat .env | grep -v '#' | awk '/=/ {print $1}')
+cd ..
+
+mvn package -DskipTests
+docker run --rm -dit -v "${ADMIN_VOLUME}":/home --name "${ADMIN_CONTAINER_NAME}" ubuntu
+docker cp ./spring-boot/proxy/target/coatrack-proxy-"${TAG}".jar "${ADMIN_CONTAINER_NAME}":/home
```

Comment:
Is it maybe possible to use a wildcard asterisk here so that the `TAG` does not need to be hard-coded?
```suggestion
docker cp ./spring-boot/proxy/target/coatrack-proxy-*.jar "${ADMIN_CONTAINER_NAME}":/home
```

Reply:
The copy of the proxy jar will be removed in #38. Please, take a look at my comment at the very bottom.